arrow-ipc/src/convert.rs

```diff
@@ -76,19 +122,9 @@ impl IpcSchemaConverter<'_> {
 }
 
 /// Serialize a schema in IPC format
-#[deprecated(
-    since = "54.0.0",
-    note = "Use `IpcSchemaConverter` instead. This function will be removed in the next release."
-)]
+#[deprecated(since = "54.0.0", note = "Use `IpcSchemaConverter`.")]
 pub fn schema_to_fb(schema: &Schema) -> FlatBufferBuilder<'_> {
-    let mut fbb = FlatBufferBuilder::new();
```

Comment:
Here and below, I removed the duplicate implementation and call into the new encoder -- so we can keep the old API the same but the code is shared